std_msgs/Header header
LineMsg[] lines
CornerMsg[] corners

